releasenotes/notes/remove-snapshot-instruction-aeae8ce4602a7ee2.yaml

```diff
@@ -0,0 +1,6 @@
+---
+upgrade:
```

Comment:
Do we actually need an upgrade release note here? This doesn't really seem like a user facing change to me. It's just internally the aer instruction used under the covers on how you get the expectation value when running on aer right?

Reply:
Thanks. Sorry, I thought the release note is required for the internal change... I'll remove it.

Reply:
Yeah, the release notes are only for user facing changes. It's how we document what's new and/or different for users coming from the previous release. I wrote a little bit about the rationale behind release notes here: https://github.com/Qiskit/qiskit-terra/blob/main/CONTRIBUTING.md#release-notes but we probably should expand to make this clearer.